src/content/api/asset-types.mdx

```diff
@@ -11,19 +11,20 @@ The following table describes the Asset Types supported for payments.
-| centrapay.nzd   | Centrapay NZD wallet                                | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪     |
-| centrapay.token | Centrapay tokens                                    | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪     |
-| epay            | [EPay NZ giftcards](https://www.giftstation.co.nz/) |                                  | `NZD`                              | ⚡️ 🧪     |
-| farmlands.nzd   | [Farmlands](https://www.farmlands.co.nz/)           | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪 💸 💼 |
-| paypal          | [PayPal](https://www.paypal.com/)                   |                                  | `USD`                         | ⚡️ 🧪 💸   |
-| quartz.nzd      | Quartz NZD asset                                    |                                  | `NZD`                         | ⚡️ 🧪     |
-| venmo           | [Venmo](https://venmo.com/)                         |                                  | `USD`                         | ⚡️ 🧪 💸   |
-| uplinkapi       | Uplink API Test asset                               |                                  | `NZD`                         | 🧪        |
-| stadius         | [Stadius](https://stadius.io)                       |                                  | `NZD` `AUD` `USD` `CAD` `EUR` | ⚡️ 🧪     |
+|    Asset Type    |                     Description                     |             Category             |          Currencies           |  Flags   |
+| :--------------- | :-------------------------------------------------- | :------------------------------- | :---------------------------- | :------- |
+| bitcoin          | [Bitcoin](https://bitcoin.org/en/)                  |                                  | `NZD` `AUD`                   | ⚡️       |
+| cca.coke         | Coke tokens                                         | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪 🚫   |
+| centrapay.nzd    | Centrapay NZD wallet                                | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪     |
+| centrapay.token  | Centrapay tokens                                    | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪     |
+| epay             | [EPay NZ giftcards](https://www.giftstation.co.nz/) |                                  | `NZD`                         | ⚡️ 🧪     |
+| farmlands.nzd    | [Farmlands](https://www.farmlands.co.nz/)           | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪 💸 💼 |
+| paypal           | [PayPal](https://www.paypal.com/)                   |                                  | `USD`                         | ⚡️ 🧪 💸   |
+| centrapay.ledger | Centrapay Ledgers                                   |                                  | `NZD`                         | ⚡️ 🧪     |
```

Comment:
Change: under Category we should include a link to the Ledger Model, same as tokens

Reply:
Yep nice one!